pdr_backend/lake/ohlcv_data_factory.py

```diff
@@ -85,8 +86,38 @@ def get_mergedohlcv_df(self) -> pl.DataFrame:
         asyncio.run(self._update_rawohlcv_files(fin_ut))
         logger.info("Update all rawohlcv files: done")
         rawohlcv_dfs = self._load_rawohlcv_files(fin_ut)
-        mergedohlcv_df = merge_rawohlcv_dfs(rawohlcv_dfs)
+        for feed in self.ss.feeds:
+            columns = [
+                "timestamp",
+                "open",
+                "high",
+                "low",
+                "close",
+                "volume",
+                "Cumulative Dollar Value",
+                "Cumulative Ticks",
+            ]
+            df = rawohlcv_dfs[str(feed.exchange)][str(feed.pair)]
+            prefix_to_threshold_type = {"vb": "volume", "db": "dollar", "tb": "tick"}
```

Comment:
I don't think this mapping is needed.
You can have a feed.thresold.get_bars(self, df_pandas) method
- retrieve the threshold type from self
- build volume/tick/dollar bars accordingly

Or if that logic shouldn't reside there, create a get_threshold_bars(df_pandas, prefix) in the lake module. Return whatever kind of bars correspond to the prefix/type given

Reply:
added a _get_threshold_bars within ohlcv_data_factory